src/components/tooltip/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ChoroplethBoundFeature } from '@nivo/geo'
+
+import * as S from './styles'
+
+interface ToolTip extends ChoroplethBoundFeature {
+  id: string
+}
+
+function Tooltip(feature: ChoroplethBoundFeature) {
+  const { id, data } = feature as ToolTip
+
+  return (
+    <S.Wrapper data-testid="tooltip">
+      <S.Title>{id}</S.Title>
+      <S.Text>expressões disponíveis: {data?.expressions || 0} </S.Text>
```

Comment:
You could use nullish coalescing instead of or, this is a recommendation when the value can return `''` or `0` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_Coalescing_Operator#assigning_a_default_value_to_a_variable)